lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -118,6 +120,13 @@ class LighthouseReportViewer {
         this._reportIsFromGist = true;
         this._replaceReportHtml(reportJson);
       }).catch(err => logger.error(err.message));
+    } else if (jsonurl) {
+      loadPromise = fetch(jsonurl).then(resp => resp.json())
```

Comment:
we decided we only want to support this if the user isn't logged in to the viewer to prevent any potential XSS vulns that may be lurking in the report

I think it'll end up looking something like

```suggestion
      const firebaseAuth = this._github.getFirebaseAuth(); // new method that needs to be added to return `this._auth`
      loadPromise = firebaseAuth.getAccessTokenIfLoggedIn()
        .then(token => token ? Promise.reject(new Error('Can only use jsonurl when not logged in')) : null)
        .then(() => fetch(jsonurl))
        .then(resp => resp.json())
```